README.md

```diff
@@ -1,2 +1,307 @@
 # ovn-k-secondary-net-dhcp
-Information on how to deploy a secondary network overlay in Kubernetes for VMs with a DHCP server workload
+Information on how to deploy a secondary network overlay in Kubernetes for VMs
+with a DHCP server workload.
+
+## Introduction
+OVN (Open Virtual Network) is a series of daemons for the Open vSwitch that
+translate virtual network configurations into OpenFlow. It provides virtual
+networking capabilities for any type of workload on a virtualized platform
+(virtual machines and containers) using the same API.
+
+OVN provides a higher-layer of abstraction than Open vSwitch, working with
+logical routers and logical switches, rather than flows. More details can be
+found in the OVN architecture man page.
+
+In this post we will create a secondary network to provide east/west
+connectivity between Openshift Virtualization Virtual Machines. For that, we
+will create an overlay network (disconnected from the nodes physical network),
+and deploy a DHCP server in a pod connected to that network, which will offer
+IP addresses for the VM workloads. Keep in mind configuring IP address
+management (IPAM) in a network attachment definition for virtual machines is
+**not** supported.
```

Comment:
Which alternative approaches exists? What is the benefit over the pod network?

Reply:
Good point. 

I'll elaborate a bit around that topic.